cmake/compiler.cmake

```diff
@@ -276,19 +269,6 @@ macro(enable_tnt_compile_flags)
         endif()
     endif()
 
-    if (CMAKE_COMPILER_IS_CLANG AND CC_HAS_WNO_UNUSED_VALUE)
-        # False-positive warnings for ({ xx = ...; x; }) macroses
-        add_compile_flags("C;CXX" "-Wno-unused-value")
-    endif()
```

Comment:
It is interesting, because we actually have such code in small and tarantool itself.

I found a reason behind this suppression: https://github.com/llvm/llvm-project/issues/14119.

According to the LLVM [release dates](https://releases.llvm.org/), the last affected clang version was 3.1 and it was hopefully fixed in clang 3.2. AFAIS, neither of currently alive (or recently EOLed) distros offer such old clang versions.

Feel free to add this information into the commit message or ignore.

Reply:
Added your remark to the commit message, thanks for the research!